awsconfigs/apps/pipeline/s3-static/sync.py

```diff
@@ -0,0 +1,423 @@
+# Copyright 2020-2021 The Kubeflow Authors
```

Comment:
is this script from the community? do you have a ref link?

Reply:
mostly from community, the rest was added by meghna